### PR TITLE
pnpm version upgrade and delete some fixme comments

### PR DESCRIPTION
**Pull request by necmettindev**

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
 nodejs 20.10.0
-pnpm 8.15.5
+pnpm 9.0.2
```
Comment (KATT) [inline .tool-versions:2]:
```suggestion
pnpm 9.0.4
```

File: package.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
   "private": true,
   "engines": {
     "node": "^20.10.0",
-    "pnpm": ">=8.15.5"
+    "pnpm": ">=9.0.0"
   },
-  "packageManager": "pnpm@8.15.5",
+  "packageManager": "pnpm@9.0.2",
```
Comment (KATT) [inline package.json:8]:
```suggestion
  "packageManager": "pnpm@9.0.4",
```

**Review by KATT — CHANGES_REQUESTED** [review]
Bump pnpm 🙏 

Also make sure to allow edits from contributors when you did PRs. I tried to push to your branch but wasn't allowed

**Comment by juliusmarminge** [comment]
> Also make sure to allow edits from contributors when you did PRs. I tried to push toyour branch but wasn't allowed

That's because he opened PR from his `next` branch. You must branch out from the main branch for maintainers to be able to push

**Pull request closed without merging**